code/__DEFINES/jobs.dm

```diff
@@ -14,45 +14,46 @@
 #define JOB_DISPLAY_ORDER_ASSISTANT 1
 #define JOB_DISPLAY_ORDER_CAPTAIN 2
 #define JOB_DISPLAY_ORDER_HEAD_OF_PERSONNEL 3
-#define JOB_DISPLAY_ORDER_BARTENDER 4
-#define JOB_DISPLAY_ORDER_BOTANIST 5
-#define JOB_DISPLAY_ORDER_COOK 6
-#define JOB_DISPLAY_ORDER_JANITOR 7
-#define JOB_DISPLAY_ORDER_CLOWN 8
-#define JOB_DISPLAY_ORDER_MIME 9
-#define JOB_DISPLAY_ORDER_CURATOR 10
-#define JOB_DISPLAY_ORDER_LAWYER 11
-#define JOB_DISPLAY_ORDER_CHAPLAIN 12
-#define JOB_DISPLAY_ORDER_PSYCHOLOGIST 13
-#define JOB_DISPLAY_ORDER_AI 14
-#define JOB_DISPLAY_ORDER_CYBORG 15
-#define JOB_DISPLAY_ORDER_CHIEF_ENGINEER 16
-#define JOB_DISPLAY_ORDER_STATION_ENGINEER 17
-#define JOB_DISPLAY_ORDER_ATMOSPHERIC_TECHNICIAN 18
-#define JOB_DISPLAY_ORDER_QUARTERMASTER 19
-#define JOB_DISPLAY_ORDER_CARGO_TECHNICIAN 20
-#define JOB_DISPLAY_ORDER_SHAFT_MINER 21
-#define JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER 22
-#define JOB_DISPLAY_ORDER_MEDICAL_DOCTOR 23
-#define JOB_DISPLAY_ORDER_PARAMEDIC 24
-#define JOB_DISPLAY_ORDER_CHEMIST 25
-#define JOB_DISPLAY_ORDER_VIROLOGIST 26
-#define JOB_DISPLAY_ORDER_RESEARCH_DIRECTOR 27
-#define JOB_DISPLAY_ORDER_SCIENTIST 28
-#define JOB_DISPLAY_ORDER_ROBOTICIST 29
-#define JOB_DISPLAY_ORDER_GENETICIST 30
-#define JOB_DISPLAY_ORDER_EXP_CORPS 31 //SKYRAT EDIT ADDITON
-#define JOB_DISPLAY_ORDER_HEAD_OF_SECURITY 32
-#define JOB_DISPLAY_ORDER_WARDEN 33
-#define JOB_DISPLAY_ORDER_SECURITY_SERGEANT 34 //SKYRAT EDIT ADDITON
-#define JOB_DISPLAY_ORDER_DETECTIVE 35
-#define JOB_DISPLAY_ORDER_SECURITY_OFFICER 36
-#define JOB_DISPLAY_ORDER_JUNIOR_SECURITY_OFFICER 37
-#define JOB_DISPLAY_ORDER_SECURITY_MEDIC 38 //SKYRAT EDIT ADDITON
-#define JOB_DISPLAY_ORDER_PRISONER 39
-#define JOB_DISPLAY_ORDER_BRIGOFF 40 //SKYRAT EDIT ADDITON
-#define JOB_DISPLAY_ORDER_NANOTRASEN_REPRESENTATIVE 41 //SKYRAT EDIT ADDITON - CENTRAL ROLES BELOW HERE!
-#define JOB_DISPLAY_ORDER_BLUESHIELD 42
+#define JOB_DISPLAY_ORDER_COMMAND_SECRETARY 4
```

Comment:
Minor thing but since it was added to the other lines, don't forget to identify it as being an Skyrat edit
```suggestion
#define JOB_DISPLAY_ORDER_COMMAND_SECRETARY 4 //SKYRAT EDIT ADDITON
```